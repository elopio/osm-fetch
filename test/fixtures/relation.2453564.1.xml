<?xml version="1.0" encoding="UTF-8"?>
<osm version="0.6" generator="OpenStreetMap server" copyright="OpenStreetMap and contributors" attribution="http://www.openstreetmap.org/copyright" license="http://opendatacommons.org/licenses/odbl/1-0/">
  <relation id="2453564" changeset="13368155" timestamp="2012-10-05T03:00:08Z" version="1" visible="true" user="Bleuet Mapper" uid="762332">
    <member type="way" ref="184246400" role="outer"/>
    <member type="way" ref="184245798" role="inner"/>
    <tag k="natural" v="wetland"/>
    <tag k="source" v="NRCan-CanVec-10.0"/>
    <tag k="type" v="multipolygon"/>
  </relation>
</osm>
