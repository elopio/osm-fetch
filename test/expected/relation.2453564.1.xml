<?xml version='1.0' encoding='UTF-8'?><osm version="0.6" generator="osm-fetch"><node id="1947033346" changeset="13368155" timestamp="2012-10-05T02:46:50Z" version="1" visible="true" user="Bleuet Mapper" uid="762332" lat="48.9187291" lon="-71.6480845"/><node id="1947033349" changeset="13368155" timestamp="2012-10-05T02:46:58Z" version="1" visible="true" user="Bleuet Mapper" uid="762332" lat="48.9187511" lon="-71.6482471"/><node id="1947033363" changeset="13368155" timestamp="2012-10-05T02:46:58Z" version="1" visible="true" user="Bleuet Mapper" uid="762332" lat="48.9188905" lon="-71.6477712"/><node id="1947033368" changeset="13368155" timestamp="2012-10-05T02:46:59Z" version="1" visible="true" user="Bleuet Mapper" uid="762332" lat="48.9189273" lon="-71.6477643"/><node id="1947033373" changeset="13368155" timestamp="2012-10-05T02:46:59Z" version="1" visible="true" user="Bleuet Mapper" uid="762332" lat="48.9190028" lon="-71.6486702"/><node id="1947033383" changeset="13368155" timestamp="2012-10-05T02:46:59Z" version="1" visible="true" user="Bleuet Mapper" uid="762332" lat="48.9191202" lon="-71.6487916"/><node id="1947033389" changeset="13368155" timestamp="2012-10-05T02:46:59Z" version="1" visible="true" user="Bleuet Mapper" uid="762332" lat="48.9191932" lon="-71.6475215"/><node id="1947033395" changeset="13368155" timestamp="2012-10-05T02:47:00Z" version="1" visible="true" user="Bleuet Mapper" uid="762332" lat="48.9192319" lon="-71.6488141"/><node id="1947033418" changeset="13368155" timestamp="2012-10-05T02:47:01Z" version="1" visible="true" user="Bleuet Mapper" uid="762332" lat="48.9195191" lon="-71.6489973"/><node id="1947033422" changeset="13368155" timestamp="2012-10-05T02:47:01Z" version="1" visible="true" user="Bleuet Mapper" uid="762332" lat="48.9195811" lon="-71.6474559"/><node id="1947033434" changeset="13368155" timestamp="2012-10-05T02:47:01Z" version="1" visible="true" user="Bleuet Mapper" uid="762332" lat="48.9197508" lon="-71.6473276"/><node id="1947033443" changeset="13368155" timestamp="2012-10-05T02:47:02Z" version="1" visible="true" user="Bleuet Mapper" uid="762332" lat="48.9198889" lon="-71.6481385"/><node id="1947033445" changeset="13368155" timestamp="2012-10-05T02:47:02Z" version="1" visible="true" user="Bleuet Mapper" uid="762332" lat="48.9198979" lon="-71.6480601"/><node id="1947033446" changeset="13368155" timestamp="2012-10-05T02:47:02Z" version="1" visible="true" user="Bleuet Mapper" uid="762332" lat="48.9199089" lon="-71.6490913"/><node id="1947033451" changeset="13368155" timestamp="2012-10-05T02:47:02Z" version="1" visible="true" user="Bleuet Mapper" uid="762332" lat="48.9199367" lon="-71.6482129"/><node id="1947033454" changeset="13368155" timestamp="2012-10-05T02:47:02Z" version="1" visible="true" user="Bleuet Mapper" uid="762332" lat="48.9199909" lon="-71.6479328"/><node id="1947033457" changeset="13368155" timestamp="2012-10-05T02:47:02Z" version="1" visible="true" user="Bleuet Mapper" uid="762332" lat="48.9200477" lon="-71.6483089"/><node id="1947033463" changeset="13368155" timestamp="2012-10-05T02:47:03Z" version="1" visible="true" user="Bleuet Mapper" uid="762332" lat="48.9201187" lon="-71.647829"/><node id="1947033464" changeset="13368155" timestamp="2012-10-05T02:47:03Z" version="1" visible="true" user="Bleuet Mapper" uid="762332" lat="48.9201225" lon="-71.6490982"/><node id="1947033465" changeset="13368155" timestamp="2012-10-05T02:47:03Z" version="1" visible="true" user="Bleuet Mapper" uid="762332" lat="48.9201258" lon="-71.6468311"/><node id="1947033473" changeset="13368155" timestamp="2012-10-05T02:47:03Z" version="1" visible="true" user="Bleuet Mapper" uid="762332" lat="48.92022" lon="-71.6482668"/><node id="1947033474" changeset="13368155" timestamp="2012-10-05T02:47:03Z" version="1" visible="true" user="Bleuet Mapper" uid="762332" lat="48.9202219" lon="-71.6476664"/><node id="1947033484" changeset="13368155" timestamp="2012-10-05T02:47:04Z" version="1" visible="true" user="Bleuet Mapper" uid="762332" lat="48.9202807" lon="-71.6476322"/><node id="1947033486" changeset="13368155" timestamp="2012-10-05T02:47:04Z" version="1" visible="true" user="Bleuet Mapper" uid="762332" lat="48.9202988" lon="-71.6467155"/><node id="1947033489" changeset="13368155" timestamp="2012-10-05T02:47:04Z" version="1" visible="true" user="Bleuet Mapper" uid="762332" lat="48.9203116" lon="-71.6480964"/><node id="1947033493" changeset="13368155" timestamp="2012-10-05T02:47:04Z" version="1" visible="true" user="Bleuet Mapper" uid="762332" lat="48.92034" lon="-71.6477144"/><node id="1947033495" changeset="13368155" timestamp="2012-10-05T02:47:04Z" version="1" visible="true" user="Bleuet Mapper" uid="762332" lat="48.9203465" lon="-71.6479299"/><node id="1947033498" changeset="13368155" timestamp="2012-10-05T02:47:04Z" version="1" visible="true" user="Bleuet Mapper" uid="762332" lat="48.9203588" lon="-71.6467243"/><node id="1947033513" changeset="13368155" timestamp="2012-10-05T02:47:05Z" version="1" visible="true" user="Bleuet Mapper" uid="762332" lat="48.9204614" lon="-71.6466352"/><node id="1947033515" changeset="13368155" timestamp="2012-10-05T02:47:05Z" version="1" visible="true" user="Bleuet Mapper" uid="762332" lat="48.920471" lon="-71.6489533"/><node id="1947033543" changeset="13368155" timestamp="2012-10-05T02:47:06Z" version="1" visible="true" user="Bleuet Mapper" uid="762332" lat="48.9207538" lon="-71.6466107"/><node id="1947033572" changeset="13368155" timestamp="2012-10-05T02:47:07Z" version="1" visible="true" user="Bleuet Mapper" uid="762332" lat="48.9209093" lon="-71.6467684"/><node id="1947033605" changeset="13368155" timestamp="2012-10-05T02:47:08Z" version="1" visible="true" user="Bleuet Mapper" uid="762332" lat="48.9210545" lon="-71.6469614"/><node id="1947033614" changeset="13368155" timestamp="2012-10-05T02:47:08Z" version="1" visible="true" user="Bleuet Mapper" uid="762332" lat="48.9210919" lon="-71.648211"/><node id="1947033638" changeset="13368155" timestamp="2012-10-05T02:47:09Z" version="1" visible="true" user="Bleuet Mapper" uid="762332" lat="48.9211571" lon="-71.6472522"/><node id="1947033641" changeset="13368155" timestamp="2012-10-05T02:47:09Z" version="1" visible="true" user="Bleuet Mapper" uid="762332" lat="48.9211752" lon="-71.6474755"/><node id="1947033644" changeset="13368155" timestamp="2012-10-05T02:47:09Z" version="1" visible="true" user="Bleuet Mapper" uid="762332" lat="48.9211894" lon="-71.6479495"/><node id="1947033651" changeset="13368155" timestamp="2012-10-05T02:47:10Z" version="1" visible="true" user="Bleuet Mapper" uid="762332" lat="48.9212139" lon="-71.6476283"/><way id="184245798" visible="true" version="1" changeset="13368155" timestamp="2012-10-05T02:55:52Z" user="Bleuet Mapper" uid="762332"><nd ref="1947033443"/><nd ref="1947033445"/><nd ref="1947033454"/><nd ref="1947033463"/><nd ref="1947033474"/><nd ref="1947033484"/><nd ref="1947033493"/><nd ref="1947033495"/><nd ref="1947033489"/><nd ref="1947033473"/><nd ref="1947033457"/><nd ref="1947033451"/><nd ref="1947033443"/></way><way id="184246400" changeset="13368155" timestamp="2012-10-05T02:57:29Z" version="1" visible="true" user="Bleuet Mapper" uid="762332"><nd ref="1947033349"/><nd ref="1947033373"/><nd ref="1947033383"/><nd ref="1947033395"/><nd ref="1947033418"/><nd ref="1947033446"/><nd ref="1947033464"/><nd ref="1947033515"/><nd ref="1947033614"/><nd ref="1947033644"/><nd ref="1947033651"/><nd ref="1947033641"/><nd ref="1947033638"/><nd ref="1947033605"/><nd ref="1947033572"/><nd ref="1947033543"/><nd ref="1947033513"/><nd ref="1947033498"/><nd ref="1947033486"/><nd ref="1947033465"/><nd ref="1947033434"/><nd ref="1947033422"/><nd ref="1947033389"/><nd ref="1947033368"/><nd ref="1947033363"/><nd ref="1947033346"/><nd ref="1947033349"/></way><relation id="2453564" changeset="13368155" timestamp="2012-10-05T03:00:08Z" version="1" visible="true" user="Bleuet Mapper" uid="762332"><member type="way" ref="184246400" role="outer"/><member type="way" ref="184245798" role="inner"/><tag k="natural" v="wetland"/><tag k="source" v="NRCan-CanVec-10.0"/><tag k="type" v="multipolygon"/></relation></osm>